cliquet/storage/postgresql/schema.sql

```diff
@@ -26,13 +30,53 @@ CREATE TABLE IF NOT EXISTS records (
 
     PRIMARY KEY (id, parent_id, collection_id)
 );
+--
+-- CREATE INDEX IF NOT EXISTS will be available in PostgreSQL 9.5
+-- http://www.postgresql.org/docs/9.5/static/sql-createindex.html
+DO $$
+BEGIN
 
-DROP INDEX IF EXISTS idx_records_parent_id_collection_id_last_modified;
-CREATE UNIQUE INDEX idx_records_parent_id_collection_id_last_modified
+  IF NOT EXISTS (
+    SELECT 1 FROM pg_indexes
+       WHERE indexname = 'idx_records_parent_id_collection_id_last_modified'
+       AND tablename = 'records'
+  ) THEN
+
+  CREATE UNIQUE INDEX idx_records_parent_id_collection_id_last_modified
     ON records(parent_id, collection_id, last_modified DESC);
```

Comment:
What do you mean exactly, by now if the index is not exists it will be created, and if it exists nothing will happen. I fixed an error in https://github.com/rodo/cliquet/commit/1477c25e6c8cbeefb0951a9a86a593dccac4e694 suppose it's related on what you mean.